apps/condo/domains/contact/components/contactRoles/ContactRolesSettingsContent.tsx

```diff
@@ -63,6 +66,24 @@ export const ContactRolesSettingsContent = (props) => {
         skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
     })
 
+    const sortedRoles = useMemo(() => {
+        const sorterByName = sorters.reduce((sorter, next) => {
+            if (next.columnKey === 'name') {
+                return next
+            }
+            return null
+        }, null)
+
+        if (sorterByName && roles.length > 0) {
```

Comment:
`if (!isEmpty(sorterByName) && !isEmpty(roles)) {`

Reply:
`sorterByName` is always null or an object, see reducer function.

`roles` is always an array.
<img width="949" alt="image" src="https://user-images.githubusercontent.com/25384290/181180224-4e807275-c27e-4564-80a3-3ae725ef831c.png">